projects/goharbor/harbor/docker/linux/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG RELEASE_BRANCH
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install -y shadow-utils >> /dev/null \
+    && yum clean all \
+    && groupadd -f -r -g 10000 chart \
+    && useradd --no-log-init -m -g 10000 -u 10000 chart
+
+COPY _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksa/helm/chartmuseum/chartm /home/chart
+COPY ../harbor/make/photon/chartserver/docker-entrypoint.sh /home/chart/
+COPY ../harbor/make/photon/common/install_cert.sh /home/chart/
+COPY _output/LICENSES /LICENSES
+COPY ATTRIBUTION.txt /ATTRIBUTION.txt
+
+RUN chown -R chart:chart /etc/pki/tls/certs \
```

Comment:
i think we for the stuff in home/chart you could add the --chown the COPY command above.